src/config/musl.md

```diff
@@ -0,0 +1,99 @@
+# musl
+
+[musl](https://musl.libc.org/) is a libc implementation which strives to be
+lightweight, fast, simple, and correct.
+
+Void officially supports musl by using it in its codebase for all target
+platforms except i686. Additionally, all the packages in our official
+repositories are available with musl-linked binaries in addition to their glibc
+counterparts.
+
+Currently, there are nonfree and debug subrepositories for musl, but no multilib
+subrepo.
+
+Generally speaking, a musl based system is a good choice for servers, embedded
+solutions and old PCs, but it may not be worth the effort to use on a modern
+desktop computer.
+
+## Possible problems
+
+Some programs (mostly graphical applications) will work incorrectly, or
+segfault, when run under musl. This may be due to programs expecting some
+glibc-specific behavior. However, currently most breakages are caused by default
+stack size differences - try running the program from a shell that has called
+`ulimit -s hard`.
```

Comment:
i would just leave the entire paragraph out in favor of something more vague along the lines of "In some cases software compatibility may be worse with musl". Crashing is not expected behavior, and most of the stack size issues have been ironed out anyway.

Reply:
Ah okay - PR updated.